core/discovery/discovery.go

```diff
@@ -45,6 +45,8 @@ const (
 const (
 	// ProposalEventTopic represent proposal events topic.
 	ProposalEventTopic = "proposalEvent"
+	// IdentityRegistrationTopic represent topics for identity registration events.
+	IdentityRegistrationTopic = "identityRegistrationOccured"
```

Comment:
```suggestion
	IdentityRegistrationTopic = "identityRegistrationOccurred"
```

Reply:
Just for the feature. As go doesn't have  enums usually you name related const starting with a type.

```go
const (
   TopicProposalEvent = ".."
   TopicIdentityRegistration = ".."
   TopicAnotherOne = ".."
)
```
Some examples from go std https://github.com/golang/net/blob/master/http2/frame.go#L30